.utility/update-docs-repo.sh

```diff
@@ -11,7 +11,7 @@ git subtree --prefix=docs/ split -b documentation
 
 cd $HOME
 
-git clone --quiet --branch=master git@github.com:loklak/dev.loklak.org.git loklak_docs
```

Comment:
wouldn't replacing this line by just `git clone --quiet --branch=master https://github.com:loklak/dev.loklak.org.git loklak_docs` solve the issue?
what is the use of `$LOKLAK_DEV_API_KEY`?